packages/create-frigg-app/README.md

```diff
@@ -1,7 +1,15 @@
-# create-frigg-app
+# Frigg App Template
 
-This package includes the global command for [Create Frigg App](https://github.com/friggframework/create-frigg-app).<br>
-Please refer to its documentation:
+Welcome to the Frigg App Template! This template helps you quickly set up a new application using the Frigg framework. For the source code and additional resources, visit our [GitHub Repository](https://github.com/friggframework/create-frigg-app).
```

Comment:
I would keep the text about global command for Create Frigg App, and add some notes about how it leverages ready-made templates to help you quickly set up new applications using the Frigg framework